src/libraries/grommet/components.js

```diff
@@ -2050,37 +2091,57 @@ export const components = {
   DataFilters: {
     component: DataFilters,
     name: 'DataFilters',
+    documentation: 'https://v2.grommet.io/datafilters',
     defaultProps: {
       drop: true,
     },
     properties: {
       drop: true,
+      heading: '',
       layer: false,
+      clearFilters: true,
+      updateOn: ['change', 'submit'],
     },
   },
   DataFilter: {
     component: DataFilter,
     name: 'DataFilter',
-    properties: {},
+    documentation: 'https://v2.grommet.io/datafilter',
+    properties: {
+      property: '',
+    },
   },
   DataSearch: {
     component: DataSearch,
     name: 'DataSearch',
-    properties: {},
+    documentation: 'https://v2.grommet.io/datasearch',
+    defaultProps: {
```

Comment:
Can you update this to the following? The defaults were out of sync with grommet behavior:
```
    properties: {
      drop: false,
      responsive: false,
    },
```

Reply:
You can remove the "defaultProps" section.

Reply:
@Sulaymon333 I don't see this comment incorporated.